CMakeLists.txt

```diff
@@ -7,7 +7,14 @@ else()
   cmake_minimum_required(VERSION 3.10)
 endif()
 
-project(bento4)
+function(GET_BENTO4_VERSION)
```

Comment:
Why use uppercase here?

Reply:
I have aways done CMake functions like that. 😄 Honestly I have no idea, I think I'm just cargo-culting an historic CMake style quirk. It can definitely be made lower case if you would rather.

I just ran `grep -nr "function(" /usr/local/share/cmake` and it's a bit of a mix of styles.

Reply:
Yeah, there's no consistent upper/lowercase norm with CMake. I was just surprised that in this diff, the function is declared with uppercase, but then called as lowercase a few lines below. Not a problem, just a cosmetic issue.